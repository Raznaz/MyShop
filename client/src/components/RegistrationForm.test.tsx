import { render, screen } from '@testing-library/react';
import RegistrationForm from './RegistrationForm';

describe('component Registration form', () => {
  it('should ', () => {
    render(<RegistrationForm />);
    screen.getByText(/text/i);
  });
});
