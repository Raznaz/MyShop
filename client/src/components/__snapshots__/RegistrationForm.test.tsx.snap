// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component Registration form should  1`] = `
<DocumentFragment>
  <div>
    <h1>
      Registration
    </h1>
    <input
      placeholder="Email"
      type="text"
      value=""
    />
    <input
      placeholder="Password"
      type="password"
      value=""
    />
    <button>
      Registration
    </button>
  </div>
</DocumentFragment>
`;
